pages/[org].js

```diff
@@ -118,16 +126,15 @@ function Org({ org: orgRes, errorCode, repos, balance, ...props }) {
 
 Org.getInitialProps = async ctx => {
   try {
-    console.log(process.env)
     const org = await request
       .get(`https://api.github.com/orgs/${ctx.query.org}`)
       .set('User-Agent', 'Delta')
-      .auth('ee508729e6002c32d53b', 'e8ed912f2b6fdcccbef5aecfcfb23a1d4b3dea13')
+      .auth(GITHUB_CLIENT_ID, GITHUB_CLIENT_SECRET)
```

Comment:
did we roll these?

Reply:
Roll? like delete?

I cleated a new oauth app in the flowerpot org, and those keys were not committed to github.